config/config.go

```diff
@@ -68,6 +68,7 @@ type PluginConfigs map[string]PluginConfig
 type PluginConfig struct {
 	Command              string
 	NotificationInterval *int32  `toml:"notification_interval"`
+	ExecutionInterval    *int32  `toml:"execution_interval"`
```

Comment:
ExecutionInterval doesn't work in metric plugins.
It's OK. I think it is preferred behaviour.
Please comment it.